CONTRIBUTING.md

```diff
@@ -1,43 +1,25 @@
 Please work off the develop branch and make pull requests to that branch. The master branch will only be updated for new releases.
 
-The Bitshares UI team is supported by this [worker proposal](https://www.bitshares.foundation/workers/2018-08-bitshares-ui). It provides the funds needed to pay the coordinator and the bounties and the Bitshares Foundation.
+The Bitshares UI team is supported by this [worker proposal](https://www.bitshares.foundation/workers/2019-02-bitshares-ui). It provides the funds needed to pay for the development team.
```

Comment:
"...the development team and community bounties." ?

Reply:
right.... bounties we have 🥇